kubetest/kops.go

```diff
@@ -530,6 +531,40 @@ func (k kops) TestSetup() error {
 	return nil
 }
 
+// BuildTester returns a standard ginkgo-script tester, unless KUBETEST_RUN_GINKGO_DIRECT=y, when we build an e2e.Tester
+func (k kops) BuildTester(o *options) (Tester, error) {
```

Comment:
Why this should live in kops.go? Looks like it can be shared across multiple deployers..

Reply:
This is building the settings that kops sets for calling ginkgo.  My opinion is that we should not share this, even if we end up with similar implementations in each.  The reason is that we have a shared implementation today, in bash, and it is hard to make the changes needed in kops, and IIRC the bash has lots of conditionals to enable it to be used elsewhere.

I'd suggest it's probably simpler to just go with simpler explicit logic, even at the cost of duplicating.  If we then decide to pull out a common "initialize" function, we could do that - but I suspect it won't be worth it.